systemvm/patches/debian/buildsystemvm.sh

```diff
@@ -228,7 +228,7 @@ cat > etc/init.d/iptables-persistent << EOF
 #!/bin/sh
 ### BEGIN INIT INFO
 # Provides:          iptables
-# Required-Start:    mountkernfs $local_fs
+# Required-Start:    mountkernfs $local_fs cloud-early-init
```

Comment:
@PaulAngus the buildsystemvm.sh is an old script, this can be actually removed, see the comment readme/docs.